pom.xml

```diff
@@ -65,6 +65,7 @@
     <slf4j.version>1.7.30</slf4j.version>
     <protobuf.version>2.5.0</protobuf.version>
     <roaringbitmap.version>0.7.45</roaringbitmap.version>
+    <snappy-java.version>1.1.8.4</snappy-java.version>
```

Comment:
@iwasakims If you have a use case, and if you feel this would help, we can bump it to 1.1.10.4 like in HADOOP-18912 to be in sync with Hadoop & merge